src/main/java/org/folio/services/transactions/BaseTransactionService.java

```diff
@@ -50,4 +59,35 @@ public void validateTransactionType(Transaction transaction, Transaction.Transac
     }
   }
 
+  public CompletableFuture<List<Transaction>> retrieveFromTransactions(List<String> groupFundIds, String fiscalYearId,
```

Comment:
Rename 1st parameter of new methods from `groupFundIds`,  `ledgetFundIds` to `fundIds `